src/Exception/InvalidConfigException.php

```diff
@@ -5,11 +5,16 @@
 use LogicException;
 use TomPHP\ExceptionConstructorTools;
 
+/**
+ * @api
```

Comment:
@localheinz Do you mean the mixing of the `@api` on the class and the `@internal` on the the methods? My reasoning is that people should be catching the exceptions but not creating them.

Or are you saying we should not use the `@api` tag at all and only use `@internal`? In this case how to you feel about blacklisting on the methods in exceptions (for the reason above)?